cognite/neat/_issues/warnings/_general.py

```diff
@@ -27,3 +27,10 @@ class RegexViolationWarning(NeatWarning):
     identifier: str
     pattern_name: str
     motivation: str | None = None
+
+
+@dataclass(unsafe_hash=True)
+class MissingCogniteClientWarning(NeatWarning):
```

Comment:
This is used in the importer, and caught by the session that raises an appropriate error.